### PR TITLE
Day3

### PR DESCRIPTION
**Pull request by andreeaNeamtiu**

### CODE REVIEW

File: evaluations.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 			<th>Nume</th>
 			<th>Tehnologie</th>
 			<th>Nivel</th>
-			<th></th>
+			<th class="table-heading-details"></th>
```
Comment (softvision-alexcarcea) [inline evaluations.html:26]: Why is this class needed for the very last column, only?

File: style.css [hunk 2]
```diff
@@ -10,24 +10,24 @@ body
...
 }
 
 nav a
 {
 	color: #636262;
-	width: 10%;
-	padding-right: 40px;
-	padding-left: 40px;
-	/*float: left;*/
+	padding: 10px 50px 10px 50px;
```
Comment (softvision-alexcarcea) [inline style.css]: Can you do this in a more succinct way?

**Review by softvision-alexcarcea — APPROVED** [review]
There's too little in this PR that I can pick on... next time, give me something to keep me busy, will ya? :P

File: newEvaluation.html [hunk 3]
```diff
@@ -17,16 +17,16 @@
 	
 	<!-- Section1 -->
 	<div class = "fieldset-container">
-	<form>
+	<form method=POST action=action.php>
```
Comment (softvision-alexcarcea) [inline newEvaluation.html]: It is a best practice to always put quotes around attribute values for consistency and extra safety. This _is_ going to work as it is now, but it's still good to get in the habit of doing things consistently. :wink:

**Review by softvision-alexcarcea — APPROVED** [review]

**Review by gcazaciuc — APPROVED** [review]

File: evaluations.html (hunk 1 above)
Comment (andreeaNeamtiu) [reply evaluations.html:26]: It seems to me that the last column is smaller than the others.

**Pull request merged**